remoting/getty/config.go

```diff
@@ -199,6 +213,17 @@ func (c *ClientConfig) CheckValidity() error {
 func (c *ServerConfig) CheckValidity() error {
 	var err error
 
+	if len(c.HeartbeatPeriod) == 0 {
+		c.heartbeatPeriod = 15 * time.Second
```

Comment:
ur default timeout is 5s while hb period is 15s. very strange.

Reply:
above